tools/project_manager/show_word_context.php

```diff
@@ -28,115 +27,93 @@
 
 // get the correct layout
 $userSettings =& Settings::get_Settings($pguser);
-// if not set gives LAYOUT_HORIZ
-$default_layout =  $userSettings->get_value("show_word_context_layout", LAYOUT_HORIZ);
-
-$layout_choices = array(LAYOUT_HORIZ, LAYOUT_VERT);
-$layout = get_enumerated_param($_GET, 'layout', $default_layout, $layout_choices);
-if($layout != $default_layout)
-{
-    $userSettings->set_value("show_word_context_layout", $layout);
-}
+// if not set gives "horizontal"
+$default_layout =  $userSettings->get_value("show_word_context_layout", "horizontal");
```

Comment:
If we're never setting in the DB (which is fine) we should ditch using this so that we're only using local storage and not a mix of the two. Here and in `tools/project_manager/show_good_word_suggestions_detail.php`.